tasks/level-1/5.4.4.yml

```diff
@@ -4,9 +4,9 @@
 # 5.4.4 Ensure default user umask is 027 or more restrictive
 
 - name: 5.4.4 - Ensure default user umask is 027 or more restrictive
-  lineinfile:
-    regexp: "^umask\\s+"
-    line: "umask {{ cis_umask_default }}"
+  replace:
+    regexp: "umask\\s+\\d+"
```

Comment:
Should this retain the `^`?

Reply:
Here is what current content of /etc/bashrc and /etc/profile looks like

>if [ $UID -gt 199 ] && [ "`/usr/bin/id -gn`" = "`/usr/bin/id -un`" ]; then
> &nbsp;&nbsp;&nbsp;    umask 002
>else
> &nbsp;&nbsp;&nbsp;     umask 022
> fi

We do not want to change the IF part as it is used by system users and may cause issues.

So we should actually search for "\\s*umask\\s+022" as there is space before "umask" and there can be one or more before "022".